Upstream-Status: backport

Dropped ChangeLog part as it does not apply.

From 2abe9f1507f971d1513a2d75a865ec4615d2d6b8 Mon Sep 17 00:00:00 2001
From: Petr Machata <pmachata@redhat.com>
Date: Thu, 6 Sep 2012 12:41:14 -0700
Subject: [PATCH] Synchronize with linux/elf.h.

Add new ARM and s390 note constants e.g. NT_*.
---
 ChangeLog | 7 +++++++
 elf/elf.h | 9 +++++++++
 2 files changed, 16 insertions(+)

diff --git a/ChangeLog b/ChangeLog
index 7385bcb..986ed54 100644
diff --git a/elf/elf.h b/elf/elf.h
index 6c85f92..41fc3bd 100644
--- a/elf/elf.h
+++ b/elf/elf.h
@@ -622,6 +622,15 @@ typedef struct
 #define NT_386_TLS	0x200		/* i386 TLS slots (struct user_desc) */
 #define NT_386_IOPERM	0x201		/* x86 io permission bitmap (1=deny) */
 #define NT_X86_XSTATE	0x202		/* x86 extended state using xsave */
+#define NT_S390_HIGH_GPRS	0x300	/* s390 upper register halves */
+#define NT_S390_TIMER	0x301		/* s390 timer register */
+#define NT_S390_TODCMP	0x302		/* s390 TOD clock comparator register */
+#define NT_S390_TODPREG	0x303		/* s390 TOD programmable register */
+#define NT_S390_CTRS	0x304		/* s390 control registers */
+#define NT_S390_PREFIX	0x305		/* s390 prefix register */
+#define NT_S390_LAST_BREAK	0x306	/* s390 breaking event address */
+#define NT_S390_SYSTEM_CALL	0x307	/* s390 system call restart data */
+#define NT_ARM_VFP	0x400		/* ARM VFP/NEON registers */
 
 /* Legal values for the note segment descriptor types for object files.  */
 
-- 
1.8.0

